runtimes/common/src/constants.rs

```diff
@@ -434,7 +434,7 @@ pub mod public_credentials {
 	use super::*;
 
 	/// The size is checked in the runtime by a test.
-	pub const MAX_PUBLIC_CREDENTIAL_STORAGE_LENGTH: u32 = 257;
+	pub const MAX_PUBLIC_CREDENTIAL_STORAGE_LENGTH: u32 = 355;
```

Comment:
is there a reason for `355` and not `357`?

Reply:
It's the result of running the unit tests for the runtimes as in https://github.com/KILTprotocol/kilt-node/blob/e95d50e7eee6f3542cd090e82930a842d5c0f0fd/runtimes/spiritnet/src/tests.rs#L106. Agree this is not very intuitive, but that's the structure we have in place right now.

Reply:
The increase in value is because we now store more information for a given credential, including CType (for filtering), and authorization info.

Reply:
ah right i remember now. 🙈 It was me who introduced this right?

Reply:
... and me who pointed out it wasn't super clear 😇